### PR TITLE
Issue/17809 moving showcase to core

### PR DESCRIPTION
**Pull request by pedrofs**
It solves #28

### CODE REVIEW

**Comment by markstory** [comment]
:clap:

File: src/Model/Table/PerspectiveImagesTable.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	 *
+	 * @return void
+	 */
+	public function initialize(array $config)
+	{
+		parent::initialize($config);
+
+        $this->addBehavior('Josegonzalez/Upload.Upload', [
+        	'file' => [
+        		'path' => 'webroot{DS}files{DS}{model}{DS}{field}{DS}{microtime}'
```
Comment (steinkel) [inline src/Model/Table/PerspectiveImagesTable.php]: is this folder already taken from the "shared" folder in deploy to keep the uploaded images from old deploys?

File: composer.json [hunk 2]
```diff
@@ -11,8 +11,11 @@
 		"cakephp/migrations": "~1.0",
 		"cakephp/plugin-installer": "*",
 		"markstory/asset_compress": "~3.0",
-		"cakedc/showcase": "dev-master",
-		"cakedc/users": "@stable"
+		"cakedc/users": "@stable",
+		"burzum/cakephp-imagine-plugin": "~2.0",
+        "muffin/tags": "dev-master",
```
Comment (steinkel) [inline composer.json]: cs

File: src/Model/Table/PerspectiveImagesTable.php (hunk 1 above)
Comment (pedrofs) [inline src/Model/Table/PerspectiveImagesTable.php]: (y)

**Pull request merged**